internal/db/db_blueprints.go

```diff
@@ -18,6 +19,23 @@ const (
 		INSERT INTO blueprint_versions(id, blueprint_id, version, body)
 		VALUES($1, $2, $3, $4)`
 
+	sqlGetBlueprintComposes = `
+		SELECT composes.job_id, blueprint_versions.version, composes.request, composes.created_at, composes.image_name, composes.client_id
+		FROM composes INNER JOIN blueprint_versions ON composes.blueprint_version_id = blueprint_versions.id
+		WHERE composes.org_id = $1
+		  	AND blueprint_versions.blueprint_id = $2
+			AND CURRENT_TIMESTAMP - composes.created_at <= $3
+			AND ($4::text[] is NULL OR composes.request->'image_requests'->0->>'image_type' <> ALL($4))
+			AND composes.deleted = FALSE
+		ORDER BY composes.created_at DESC
+		LIMIT $5 OFFSET $6`
+
+	sqlCountActiveBlueprintComposesSince = `
+		SELECT COUNT(*)
+		FROM composes INNER JOIN blueprint_versions ON composes.blueprint_version_id = blueprint_versions.id
+		WHERE org_id=$1 AND blueprint_versions.blueprint_id = $2 AND CURRENT_TIMESTAMP - created_at <= $3 AND deleted = FALSE
+		AND ($4::text[] is NULL OR request->'image_requests'->0->>'image_type' <> ALL($4))`
```

Comment:
Generic suggestion for all these long query strings with joins, you can rename tables to single char names:

```
SELECT c.job_id, c.version, c.request, c.created_at, c.image_name, c.client_id, ...
FROM composes AS c INNER JOIN blueprint_versions AS v
ON c.blueprint_version_id = v.id
```

Reply:
I find it more confusing then helpful, unless the short version is very clear.
Here where we actually work with the SQL, I prefer longer names for readability.
It's just my preference tho, if you think we should alias it, I'm happy to ;)

Reply:
Sure it is just a random thought.